codec-haproxy/src/main/java/io/netty/handler/codec/haproxy/HAProxyMessage.java

```diff
@@ -519,4 +498,93 @@ public int destinationPort() {
     public List<HAProxyTLV> tlvs() {
         return tlvs;
     }
+
+    private HAProxyMessage copyMessage(List<HAProxyTLV> tlvs, ByteBuf content) {
+        return new HAProxyMessage(protocolVersion, command, proxiedProtocol, sourceAddress,
+                                  destinationAddress, sourcePort, destinationPort, tlvs, content);
+    }
+
+    @Override
+    public HAProxyMessage copy() {
+        List<HAProxyTLV> copiedTLVs = new LinkedList<HAProxyTLV>();
```

Comment:
why not an `ArrayList` to reduce the GC ?

Reply:
Done